packages/api/cms-api/src/redirects/import-redirects.console.ts

```diff
@@ -0,0 +1,217 @@
+                                ],
+                                activeType: "external",
+                            })
+                            .transformToBlockData(),
+                    });
+
+                    success++;
+
+                    await this.repository.flush();
+                } else {
+                    const redirect = this.repository.create({
+                        id: uuid(),
+                        sourceType: RedirectSourceTypeValues.path,
+                        source: row["source"],
+                        target: this.linkBlock
+                            .blockInputFactory({
+                                attachedBlocks: [
+                                    {
+                                        type: "external",
+                                        props: {
+                                            targetUrl: row["target"],
+                                            openInNewWindow: true,
+                                        },
+                                    },
+                                ],
+                                activeType: "external",
+                            })
+                            .transformToBlockData(),
+                        active: true,
+                        generationType: RedirectGenerationType.manual,
+                        createdAt: new Date(),
+                        updatedAt: new Date(),
+                        comment,
+                        scope: row["scope"],
+                    });
+
+                    success++;
+                    this.repository.persist(redirect);
+                }
+            } else {
+                console.log(`Error for Redirect ${row["source"]}`);
+                error++;
+            }
+        }
+
+        await this.repository.flush();
+        console.log(`\nSuccess: ${success}`);
+        console.log(`Error: ${error}`);
+    }
+
+    readRedirectsCsv = async (
+        filePath: string,
+    ): Promise<
+        Array<{
+            target: string;
+            target_type: string;
+            source: string;
+            comment: string;
+            scope: Scope;
+        }>
+    > => {
+        const imports: Array<{
+            target: string;
+            target_type: string;
+            source: string;
+            comment: string;
+            scope: Scope;
+        }> = [];
+        return new Promise((resolve, reject) => {
+            fs.createReadStream(filePath)
+                .pipe(csv.parse({ headers: true, delimiter: ";" }))
+                .on("error", (error) => {
+                    console.error(error);
+                    reject(error);
+                })
+                .on("data", (row) => {
+                    const hasScope = Object.keys(row).some((key) => key.startsWith("scope_"));
+
+                    if (!hasScope) {
+                        throw Error("Could not find a scope in the passed file!");
+                    }
```

Comment:
The redirects module in Comet has an optional scope, so we need to support having no scope here.